### ISSUE TITLE
Changeset with feature counts as zero

### ISSUE THREAD

**Issue by bkowshik** [willemarcel/osmcha-django#83]
Came across a changeset which had the values for features created, modified and deleted as `0`.

- http://osmcha.mapbox.com/47185405/

<img width="602" alt="screen shot 2017-04-06 at 11 50 36 am" src="https://cloud.githubusercontent.com/assets/2899501/24740543/4d483c34-1abf-11e7-8633-559d4e415072.png">

Curious to see if there were any other changesets with similar behavior, did a [query on osmcha](http://osmcha.mapbox.com/?is_suspect=True&checked=All&modify__gte=0&is_whitelisted=True&all_reason=True&delete__gte=0&create__gte=0&modify__lte=0&create__lte=0&delete__lte=0) and found `46 changesets`. A majority of these have occurred in the last couple of days.

**Comment by batpad** [comment]
> Curious to see if there were any other changesets with similar behavior, did a query on osmcha and found 46 changesets. A majority of these have occurred in the last couple of days.

So the reason you see them only in the past 2 days is that we had a bug in `osmcha` that was dropping these changesets earlier :-( - this was fixed with https://github.com/willemarcel/osmcha-django/commit/cb24770942e72f980afdac18c63220c8782cd498

Still need to properly understand all the different reasons / scenarios where changesets are empty, but I have seen that in the past - technically, it is possible for a user to open a changeset on OSM, never add any features to it, and then close it. Am curious if these are all cases of that happening, or  ....

**Comment by willemarcel** [comment]
> So the reason you see them only in the past 2 days is that we had a bug in osmcha that was dropping these changesets earlier :-( - this was fixed with cb24770

@batpad The changeset 47185405 doesn't fit in this case. It has a bbox and I imported it here in my machine and it was created with all the count fields correctly.

Could you check the logs to see if there is some error when analysing this changeset? Maybe osmcha couldn't read https://www.openstreetmap.org/api/0.6/changeset/47185405/download when it was analysing the changeset...

**Comment by batpad** [comment]
@willemarcel I see logs like this:

```
gabbar prediction: False
47528683 created
It seems this changeset was redacted
gabbar prediction: False
It seems this changeset was redacted
gabbar prediction: False
47529991 created
gabbar prediction: False
```

The code seems to think the changeset was redacted? Going to poke and look what might trigger that :(
